Sources/SwiftSyntaxBuilder/generated/BuildableNodes.swift

```diff
@@ -17,397 +17,440 @@ import SwiftSyntax
 
 extension AccessorDeclSyntax {
   /// A convenience initializer that allows initializing syntax collections using result builders
-  public init(leadingTrivia: Trivia? = nil, unexpectedBeforeAttributes: UnexpectedNodesSyntax? = nil, attributes: AttributeListSyntax? = nil, unexpectedBetweenAttributesAndModifier: UnexpectedNodesSyntax? = nil, modifier: DeclModifierSyntax? = nil, unexpectedBetweenModifierAndAccessorKind: UnexpectedNodesSyntax? = nil, accessorKind: TokenSyntax, unexpectedBetweenAccessorKindAndParameter: UnexpectedNodesSyntax? = nil, parameter: AccessorParameterSyntax? = nil, unexpectedBetweenParameterAndEffectSpecifiers: UnexpectedNodesSyntax? = nil, effectSpecifiers: DeclEffectSpecifiersSyntax? = nil, unexpectedBetweenEffectSpecifiersAndBody: UnexpectedNodesSyntax? = nil, @CodeBlockItemListBuilder bodyBuilder: () -> CodeBlockItemListSyntax? = {
+  @_disfavoredOverload
```

Comment:
@ahoppen,  I've tried to add `@_disfavoredOverload` to the function, as we get compile errors in the test with doing 
```swift
    let carriateReturnsStruct = StructDeclSyntax(
      leadingTrivia: [
        .docLineComment("/// A nested struct"),
        .carriageReturns(1),
        .docLineComment("/// with multi line comment where the newline is a CR"),
        .carriageReturns(1),
      ],
      structKeyword: .keyword(.struct),
      identifier: "CarriateReturnsStruct"
    )
```

etc. 

If I understand it correctly is it because it chose the result builder init method instead of the "normal" one

Reply:
I don’t understand why `@_disfavoredOverload` would be necessary here. But I haven’t looked into it either. Will take a look on Monday.

Reply:
I think a it's because it cannot chose the init method that doesn't throw.

Reply:
I just debugged and I believe this is due to https://github.com/apple/swift/issues/63285. 

The gist is that `rethrows` doesn’t work property with throwing closures that have default values.

The solution would be to stamp out two copies of the initializer. One that is the one we have currently and the other one is what you generated but with `throws` instead of `rethrows` (essentially we are just implementing the `rethrows` behavior with the two overloads ourselves.